app/main.py

```diff
@@ -1,3 +1,45 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: (int, float)) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: (int, float)) -> Distance:
+        if isinstance(other, Distance):
+            return Distance(self.km + other.km)
+        return Distance(self.km + other)
+
+    def __iadd__(self, other: (int, float)) -> Distance:
+        if isinstance(other, Distance):
+            self.km += other.km
+            return self
+        self.km += other
+        return self
+
+    def __mul__(self, other: (int, float)) -> Distance:
+        return Distance(km=round(self.km * other, 3))
+
+    def __truediv__(self, other: (int, float)) -> Distance:
+        return Distance(km=round(self.km / other, 2))
+
+    def __lt__(self, other: (int, float)) -> bool:
+        return self.km < other
+
+    def __gt__(self, other: (int, float)) -> bool:
+        return self.km > other
+
+    def __eq__(self, other: (int, float)) -> bool:
+        return self.km == other
+
+    def __le__(self, other: (int, float)) -> bool:
+        return self.km <= other
+
+    def __ge__(self, other: (int, float)) -> bool:
+        return self.km >= other
```

Comment:
implement type check for this methods, what if other will be an instnace of Distance ? it will not work correctly